src/System.Management.Automation/engine/Attributes.cs

```diff
@@ -1098,6 +1098,194 @@ private static Type GetCommonType(Type minType, Type maxType)
     }
 
     /// <summary>
+    /// Base class for postive number valididation attributes
+    /// </summary>
+    /// <remarks>
+    /// Base class used by postive and non negative attribute
+    /// <seealso cref="ValidatePositiveAttribute"/>,
+    /// <seealso cref="ValidateNonNegativeAttribute"/>
+    /// </remarks>
+    [AttributeUsage(AttributeTargets.Field | AttributeTargets.Property)]
+    public class ValidatePositiveBaseAttribute : ValidateEnumeratedArgumentsAttribute
```

Comment:
This base class is not generally useful, so I'd prefer avoiding introducing a new public type.